app/uk/gov/hmrc/tariffclassificationfrontend/service/FileStoreService.scala

```diff
@@ -57,4 +60,12 @@ class FileStoreService @Inject()(connector: FileStoreConnector) {
     }
   }
 
+  def upload(f: MultipartFormData.FilePart[TemporaryFile])(implicit hc: HeaderCarrier): Future[FileStoreAttachment] = {
```

Comment:
We shouldnt be passing `MultipartFormData.FilePart`s around, thats view specific code. Just have the argument as type `TemporaryFile`

Reply:
e.g. If a Service wanted to upload a file for example, we'd have to wrap it in a HTML form model to do it